lms/static/scripts/frontend_apps/components/ValidationMessage.js

```diff
@@ -33,11 +34,13 @@ export default function ValidationMessage({
   });
 
   return (
-    // FIXME-A11Y
-    // eslint-disable-next-line jsx-a11y/no-static-element-interactions, jsx-a11y/click-events-have-key-events
-    <div onClick={closeValidationError} className={errorClass}>
+    <button
+      aria-label="Validation error"
```

Comment:
What is the reason for the `aria-label` here? This will end up replacing the presumably more specific `{message}` content. My understanding is that it is generally preferable not to override the UI text with something else unless the meaning of the UI text would be unclear for some reason.

Reply:
I think you're right, can't remember why I added it